python/lsst/ctrl/mpexec/cli/script/qgraph.py

```diff
@@ -200,7 +199,6 @@ def qgraph(  # type: ignore
         raise RuntimeError("QuantumGraph is empty.")
 
     # optionally dump some info.
-    if show:
-        f.showInfo(args, pipelineObj, qgraph)
+    show.show_graph_info(qgraph, args)
```

Comment:
Line 133 above - docstring needs an update for the new type of `show` parameter.